PowerPointLabs/PowerPointLabs/Views/NarrationsLabSettingsDialogBox.xaml.cs

```diff
@@ -0,0 +1,45 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Windows;
+
+namespace PowerPointLabs.Views
+{
+    /// <summary>
+    /// Interaction logic for NarrationsLabSettingsDialogBox.xaml
+    /// </summary>
+    public partial class NarrationsLabSettingsDialogBox
```

Comment:
The behavior of this DialogBox is kind of strange. When clicking the text, the drop-down menu will not activate. DO you think you could make it trigger when the user clicks the text?